environment_raven.yml

```diff
@@ -0,0 +1,29 @@
+name: synth_seg_py39
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9.7
+  - cudatoolkit=11.2.2
```

Comment:
Both, cudatoolkit and cudnn are explicit dependencies of tensorflow-gpu when running e.g.

```
conda install -c conda-forge --dry-run tensorflow-gpu==2.11.0
```

So unless _our_ code does not explicitly use a package, we should rely on the resolving mechanism. However, conda suggests cudatoolkit 11.8.0 on my machine and maybe you had good reason to use this specific version with regards to running it at MPCDF?

Reply:
Here I just followed the installation instruction on https://www.tensorflow.org/install/pip
Actually, let's fix it to cudatoolkit 11.6 and cudnn 8.4.1 since this is the environment currently available at our raven system.